pac2go.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+function die() {
+  echo $*
+  exit 1
+}
+which 2goarray > /dev/null || go get github.com/jteeuwen/go-bindata/...
+which 2goarray > /dev/null || die "Please install 2goarray manually, then try again"
```

Comment:
2goarray looks neat, but why didn't you just us go-bindata like we already do elsewhere?